conf/messages

```diff
@@ -75,7 +75,7 @@ start.registration.IdontHave.answer=If you don’t have these details you’ll n
 #DO YOU HAVE GOVERNMENT GATEWAY ACCOUNT
 govGateway.heading=Do you already have a Government Gateway account?
 govGateway.y=Yes
-govGateway.y.helpText=You''ll need your Government Gateway user ID and password
+govGateway.y.helpText=You''ll need your existing sign in details
```

Comment:
Can you please update all the apostrophes to be `’`. I can't remember why we have to do it, but I think it's an accessibility/gds thing